### PR TITLE
Fix inference equation and make it more readable

### LINKED ISSUE #46522: BatchNormalization inference equation in doc is incorrect
## URL(s) with the issue:

https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization

## Description of issue (what needs changing):

This documentation states the following:

> During inference (i.e. when using evaluate() or predict() or when calling the layer/model with the argument training=False (which is the default), the layer normalizes its output using a moving average of the mean and standard deviation of the batches it has seen during training. That is to say, it returns (batch - self.moving_mean) / (self.moving_var + epsilon) * gamma + beta.

This equation is incorrect, testing with the source code shows it gives the wrong output.  The correct equation that gives the right output (and matches the correct equation from literature) is:

`(batch - self.moving_mean) / sqrt(self.moving_var + epsilon) * gamma + beta.`

I.e., it is missing square-root.

Further, for more clarity (to avoid confusion for some) it would be better to write it as:

`gamma*(batch - self.moving_mean) / sqrt(self.moving_var + epsilon) + beta.`

### PR DESCRIPTION
**Pull request by Suraj-Upadhyay**
Replace the denominator in the equation with its square root.
And make the multiplication of gamma a little more readable.

Signed-off-by: Suraj Upadhyay <usuraj35@gmail.com>

FIxes #46522

### CODE REVIEW

**Comment by Suraj-Upadhyay** [comment]
@googlebot I signed it!

**Comment by jiafulow** [comment]
Hi, I believe line 57 should also be fixed.

**Comment by Suraj-Upadhyay** [comment]
> Hi, I believe line 57 should also be fixed.

Hi, @jiafulow thanks for pointing that out. It nearly slipped 😅

I have force pushed a new commit fixing the equations at both the places now.

**Review by omalleyt12 — APPROVED** [review]
Thanks for the PR! LGTM
